apps/accounts/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from apps.users.models import UserProfile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = (
+            'username', 'display_name', 'email',
+            'bio', 'deleted', 'display_collections',
+            'display_collections_fav', 'homepage',
+            'location', 'notes', 'notifycompat',
+            'notifyevents', 'occupation', 'picture_type',
```

Comment:
I don't know what a lot of these fields are, wonder if @magopian does?

We can certainly expand these out if we need to and its probably easy to push something out with minimal fields.